pom.xml

```diff
@@ -4,14 +4,21 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>jenkins</groupId>
-    <artifactId>plugin-management</artifactId>
+    <parent>
+        <groupId>org.jenkins-ci</groupId>
+        <artifactId>jenkins</artifactId>
+        <version>1.51</version>
+    </parent>
     <packaging>pom</packaging>
+    <groupId>io.jenkins.plugin-management</groupId>
+    <artifactId>jenkins-plugin-management-parent-pom</artifactId>
```

Comment:
Maybe the new artifact id should match the repository name `jenkins-plugin-installation-manager`

Reply:
I already merged this. I also thought that the artifactId had to match the repository name, but for the custom war packager it also does not match: https://github.com/jenkinsci/custom-war-packager/blob/master/pom.xml